### PR TITLE
Add the username to the url

### PR DESCRIPTION
**Pull request by fnc**
Issue #2
The enhancement was implemented using react-router-dom. 

New behaviour:
- The handler for the Load button now sets the url with the username.
- Writing the username in the URL behaves the same as using the form.

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "hx-tokenizer": "^1.0.0",
     "react": "15.4.2",
     "react-dom": "15.4.2",
-    "react-motion": "^0.4.7"
+    "react-motion": "^0.4.7",
+    "react-router-dom": "^4.1.1"
```
Comment (pomber) [inline web/package.json:11]: Make sure to add this with `yarn` so the yarn.lock file keeps updated.

**Comment by pomber** [comment]
Nice, thanks! Fix the yarn.lock and I'll merge it

File: web/package.json (hunk 1 above)
Comment (fnc) [reply web/package.json:11]: I added the yarn.lock file.

**Comment by pomber** [comment]
Cool.  
One small issue, when you enter from a url, the username in the textbox should be taken from the url too. I'll merge this anyway, can you open a new issue for that?

**Pull request merged**